xrayutilities/materials/cif.py

```diff
@@ -205,9 +224,14 @@ def floatconv(string):
                         floatconv(asplit[ay_idx]),
                         floatconv(asplit[az_idx]))
                 occ = floatconv(asplit[occ_idx]) if occ_idx else 1
-                uiso = floatconv(asplit[uiso_idx]) if uiso_idx else 0
-                biso = 8 * numpy.pi**2 * uiso
-                self.atoms.append((alabel, apos, occ, biso))
+                if uiso_idx:
+                    uiso = floatconv(asplit[uiso_idx])
+                if biso_idx:
+                    uiso = 3.*floatconv(asplit[biso_idx])/(8.*numpy.pi**2)
+                else:
+                    uiso = 0
+
+                self.atoms.append((alabel, apos, occ, uiso))
 
```

Comment:
I think this part is scientifically wrong!
Where did you find the relation you use between `uiso` and `biso`? Also I think it was always `biso` which should be appended to the atoms list... Anyhow I recently implemented also a biso parser but the equation I use is different by a factor of 3. It would be nice to clarify this.

Reply:
You're correct. No prefactor of 3!
Factor of 3 is for uiso = diag(U_ij)/3, assuming U_ij is not isotropic...